src/services/AuthorizationService.ts

```diff
@@ -0,0 +1,17 @@
+import { singleton } from 'tsyringe';
+import { AppUser } from '@Entities';
+
+@singleton()
+export class AuthorizationService {
+  /* eslint-disable-next-line @typescript-eslint/no-empty-function */
+  constructor() {}
```

Comment:
o does typescript force you to have one? I thought you could just omit the ctor

Reply:
Probably not, I'll try removing the empty ctor